test/addons/rbd-mirror/start

```diff
@@ -153,8 +153,8 @@ os.chdir(os.path.dirname(__file__))
 cluster1 = sys.argv[1]
 cluster2 = sys.argv[2]
 
-clear_blocklist(cluster1)
-clear_blocklist(cluster2)
+# clear_blocklist(cluster1)
+# clear_blocklist(cluster2)
```

Comment:
This is only for testing - if successful, I'll remove the calls and unused code.